composeApp/src/androidMain/kotlin/com/github/jetbrains/rssreader/AppActivity.kt

```diff
@@ -35,30 +50,29 @@ class AppActivity : ComponentActivity() {
                         )
                     }
                 }
-                Box(
-                    Modifier.padding(
-                        WindowInsets.systemBars
-                            .only(WindowInsetsSides.Start + WindowInsetsSides.End)
-                            .asPaddingValues()
-                    )
-                ) {
+                Box(modifier = Modifier
+                    .safeContentPadding()
+                    .fillMaxSize())
+                {
                     Scaffold(
                         scaffoldState = scaffoldState,
                         snackbarHost = { hostState ->
                             SnackbarHost(
                                 hostState = hostState,
-                                modifier = Modifier.padding(
-                                    WindowInsets.systemBars
-                                        .only(WindowInsetsSides.Bottom)
+                                modifier = Modifier.Companion.padding(
+                                    WindowInsets.Companion.systemBars
+                                        .only(WindowInsetsSides.Companion.Bottom)
```

Comment:
Lots of unnecessary `Companion.` added here, probably automatically during refactoring